var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -124,6 +124,12 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
             depends_on("pika cuda_arch={0}".format(val), when="cuda_arch={0}".format(val))
             depends_on("umpire cuda_arch={0}".format(val), when="cuda_arch={0}".format(val))
 
+    patch(
+        "https://github.com/eth-cscs/DLA-Future/pull/1063/commits/efc9c176a7a8c512b3f37d079dec8c25ac1b7389.patch?full_index=1",
```

Comment:
Why are we adding a URL for the patch, _and_ committed a patch to the repo that seems unused?